packages/graphql-codegen-cli/package.json

```diff
@@ -51,7 +51,7 @@
     "@graphql-tools/github-loader": "^7.3.6",
     "@graphql-tools/graphql-file-loader": "^7.5.0",
     "@graphql-tools/json-file-loader": "^7.4.1",
-    "@graphql-tools/load": "^7.7.1",
+    "@graphql-tools/load": "7.8.0",
```

Comment:
why is this locked down?

Reply:
Because we required to update to this exact version for performance purposes

This will be fixed automatically by Renovate soon

Reply:
👍